.github/workflows/ci.yml

```diff
@@ -18,14 +21,156 @@ jobs:
           config: '.markdownlint.yaml'
           globs: '**/README.md'
 
-  ci_macos:
-    uses: ./.github/workflows/macos.yml
+  platform_checks:
+    name: Run checks on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-24.04
+            platform: linux
+          - os: ubuntu-22.04
```

Comment:
Is there any reason to still support ubuntu-22.04? I noticed the release workflow only build and publish for 24.04, so does it make sense to keep 22.04 tested in the CI as well?